src/JWT.php

```diff
@@ -236,6 +236,7 @@ protected function getClaimsArray(JWTSubject $subject)
     {
         return array_merge(
             ['sub' => $subject->getJWTIdentifier()],
+            ['prv' => get_class($subject)],
```

Comment:
I think it might be cool to change this to be an md5.. as a cheap means of obfuscation. e.g.

`md5(get_class($subject))`

I would also actually pull this out into it's own method